packages/account/src/providers/transaction-response/getDecodedLogs.ts

```diff
@@ -8,12 +8,17 @@ import type {
   TransactionResultReceipt,
 } from './transaction-response';
 
+export type DecodedLogs<T = unknown> = {
+  logs: T[];
+  groupedLogs: Record<string, T[]>;
+};
```

Comment:
```suggestion
export type DecodedLogs<T = unknown> = {
  logs: Record<string, T[]>;
  legacyLogs: T[];
};
```
Do we eventually plan to roll out ungrouped logs, as this may make more sense?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3805#discussion_r2009930027